docker/generated/bento-server/debian10/runtime/Dockerfile

```diff
@@ -1,16 +1,28 @@
-# syntax = docker/dockerfile:1.2
+# syntax = docker/dockerfile:1.3-labs
 #
 # ===========================================
 #
-# THIS IS A GENERATED DOCKERFILE DO NOT EDIT.
+# THIS IS A GENERATED DOCKERFILE. DO NOT EDIT
 #
 # ===========================================
 
-
 ARG PYTHON_VERSION
 
-FROM bento-server:base-python$PYTHON_VERSION-debian as build_image
+FROM aarnphm/bento-server:base-python$PYTHON_VERSION-debian10 as base
+
+ENV BENTOML_VERSION=1.0.0a6
+
+RUN --mount=type=cache,mode=0777,target=/root/.cache/pip \
+    pip install bentoml==${BENTOML_VERSION} --no-cache-dir
+
+FROM base as base-amd64
+
+FROM base as base-arm64
+
+FROM base-${TARGETARCH}
+
+ARG TARGETARCH
 
-ENV BENTOML_VERSION=1.0.0a5
+ARG TARGETPLATFORM
 
-RUN pip install bentoml==${BENTOML_VERSION} --no-cache-dir
\ No newline at end of file
+RUN rm -rf /var/lib/{apt,dpkg,cache,log}
```

Comment:
I need some feedback here. For all debian images, if we do this, then users won't be able to install new packages to the container. They will only have access to python, curl, bash and bentoml